Add Epicore (#386)
nf-core/mhcquant: docs/output.md

| MS2PIP   | `spec_pearson`                         | Pearson correlation coefficient between observed and predicted b and y ion intensities of MS2 spectra; higher values indicate better agreement.                                       |
| MS2PIP   | `std_abs_diff`                         | Standard deviation of absolute differences between observed and predicted b and y ion intensities; lower values indicate better agreement.                                            |
| IM2Deep  | `ccs_observed_im2deep`                 | Observed peptide collisional cross-section (CCS) value.                                                                                                                               |
| IM2Deep  | `ccs_predicted_im2deep`                | Predicted peptide CCS value by IM2Deep.                                                                                                                                               |
| IM2Deep  | `ccs_error_im2deep`                    | Difference between observed and predicted CCS values; indicates prediction accuracy.                                                                                                  |

> | Epicore  | `entire_epitope_sequence`              | The sequences of the epicore groups to which the peptide belongs.                                                                                                                     |

Reviewer: ```suggestion
| Epicore  | `entire_epitope_sequence`              | Sequences of epicore groups to which the peptide belongs.                                                                                                                     |
```

> | Epicore  | `core_epitope_sequence`                | The core sequences of the epicore groups to which the peptide belongs.                                                                                                                |

Reviewer: ```suggestion
| Epicore  | `core_epitope_sequence`                | Core sequences of epicore groups to which the peptide belongs.                                                                                                                |
```

> | Epicore  | `proteome_occurrence`                  | The accessions and positions of the core epitopes.                                                                                                                                    |

Reviewer: ```suggestion
| Epicore  | `proteome_occurrence`                  | Protein accessions and positions of core epitopes.                                                                                                                                    |
```

> | Epicore  | `consensus_epitope_intensity`          | The sum of the intensities of all peptides belonging to the epicore group.                                                                                                            |

Reviewer: ```suggestion
| Epicore  | `consensus_epitope_intensity`          | Sum of intensities of all peptides belonging to the epicore group.                                                                                                            |
```

> | Epicore  | `relative_consensus_epitope_intensity` | The relative epicore group intensity (core_epitope_intensity divided by the total intensity in the evidence file).                                                                    |

Reviewer: ```suggestion
| Epicore  | `relative_consensus_epitope_intensity` | Relative epicore group intensity (core_epitope_intensity divided by the total intensity of peptides in output file).                                                                    |
```
